ci/scripts/run_pytest.sh

```diff
@@ -25,13 +25,33 @@ if ! set_bionemo_home; then
     exit 1
 fi
 
-python -m coverage erase
+# download Codecov CLI
+curl -Os https://cli.codecov.io/latest/linux/codecov
 
+# integrity check
+curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import # One-time step
+curl -Os https://cli.codecov.io/latest/linux/codecov
+curl -Os https://cli.codecov.io/latest/linux/codecov.SHA256SUM
+curl -Os https://cli.codecov.io/latest/linux/codecov.SHA256SUM.sig
+gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
+
+shasum -a 256 -c codecov.SHA256SUM
+sudo chmod +x codecov
```

Comment:
any way we can make this work without sudo? I worry `run_pytest.sh` will be run in some CI environment where sudo isn't available.

Reply:
Thinking out loud, you might want to make this bash script with a command like arg to optionally disable codecov

Reply:
might be easier to just install `codecov-cli` in our requirements-test.txt